inputs-optional.tf

```diff
@@ -72,3 +72,7 @@ variable "source_server_id" {
   default     = null
   description = "Source server ID for point in time restore. Only used when create mode is set to PointInTimeRestore"
 }
+variable "business_area" {
```

Comment:
there's already a variable for this `project   = "sds" # sds or cft` but business_area is probably a more appropriate name.

probably want to bin project and replace any references to where it's used or reuse project (using a function to capitalise) and then we can clean up the var name later

Reply:
sure -didn't know. that, updated now